internal/services/databricks/databricks_virtual_network_peering_resource_test.go

```diff
@@ -0,0 +1,266 @@
+  name                = "acctest-ws-%[1]d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+  sku                 = "standard"
+}
+`, data.RandomInteger, data.Locations.Primary)
+}
+
+func (DatabricksVirtualNetworkPeeringResource) update(data acceptance.TestData) string {
+	template := DatabricksVirtualNetworkPeeringResource{}.template(data)
+	return fmt.Sprintf(`
+%[2]s
+
+resource "azurerm_databricks_virtual_network_peering" "test" {
+  name                = "acctest-%[1]d"
+  resource_group_name = azurerm_resource_group.test.name
+  workspace_id        = azurerm_databricks_workspace.test.id
+
+  remote_address_space_prefixes = azurerm_virtual_network.remote.address_space
+  remote_virtual_network_id     = azurerm_virtual_network.remote.id
+
+  allow_virtual_network_access = false
+}
+
+resource "azurerm_virtual_network_peering" "remote" {
+  name                      = "to-acctest-%[1]d"
+  resource_group_name       = azurerm_resource_group.test.name
+  virtual_network_name      = azurerm_virtual_network.remote.name
+  remote_virtual_network_id = azurerm_databricks_virtual_network_peering.test.virtual_network_id
+}
+`, data.RandomInteger, template)
+}
+
+func (DatabricksVirtualNetworkPeeringResource) requiresImport(data acceptance.TestData) string {
+	template := DatabricksVirtualNetworkPeeringResource{}.basic(data)
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_databricks_virtual_network_peering" "import" {
+  name                = "acctest-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  workspace_id        = azurerm_databricks_workspace.test.id
+
+  remote_address_space_prefixes = azurerm_virtual_network.remote.address_space
+  remote_virtual_network_id     = azurerm_virtual_network.remote.id
+}
+`, template, data.RandomInteger)
+}
+
+func (DatabricksVirtualNetworkPeeringResource) basic(data acceptance.TestData) string {
+	template := DatabricksVirtualNetworkPeeringResource{}.template(data)
+	return fmt.Sprintf(`
+%[2]s
+
+resource "azurerm_databricks_virtual_network_peering" "test" {
+  name                = "acctest-%[1]d"
+  resource_group_name = azurerm_resource_group.test.name
+  workspace_id        = azurerm_databricks_workspace.test.id
+
+  remote_address_space_prefixes = azurerm_virtual_network.remote.address_space
+  remote_virtual_network_id     = azurerm_virtual_network.remote.id
+}
+
+resource "azurerm_virtual_network_peering" "remote" {
+  name                      = "to-acctest-%[1]d"
+  resource_group_name       = azurerm_resource_group.test.name
+  virtual_network_name      = azurerm_virtual_network.remote.name
+  remote_virtual_network_id = azurerm_databricks_virtual_network_peering.test.virtual_network_id
+}
+`, data.RandomInteger, template)
+}
+
+func (DatabricksVirtualNetworkPeeringResource) complete(data acceptance.TestData) string {
+	template := DatabricksVirtualNetworkPeeringResource{}.template(data)
```

Comment:
we can give this method a name of the type and call the other method directly fwiw:

```suggestion
func (r DatabricksVirtualNetworkPeeringResource) complete(data acceptance.TestData) string {
	template := r.template(data)
```

Reply:
Fixed.